app/main.py

```diff
@@ -1 +1,36 @@
-# write your code here
+class Animal:
+
+    alive = []
+
+    def __init__(self,
+                 name: str,
+                 health: int = 100,
+                 hidden: bool = False) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (f"{{Name: {self.name}, "
+                f"Health: {self.health}, "
+                f"Hidden: {self.hidden}}}")
+
+    @staticmethod
+    def is_dead(lunch_name: str) -> None:
+        if lunch_name.health <= 0:
+            Animal.alive.remove(lunch_name)
+
+
+class Herbivore(Animal):
+
+    def hide(self) -> None:
+        self.hidden = not self.hidden
+
+
+class Carnivore(Animal):
+
+    def bite(self, lunch_name: str) -> None:
```

Comment:
first: do u even use this `self` ? consider making this method as static.
second: lunch_name is a bad variable name, and it is not a `str`